worlds/yachtdice/Options.py

```diff
@@ -0,0 +1,331 @@
+from dataclasses import dataclass
+
+from Options import Choice, OptionGroup, PerGameCommonOptions, Range
+
+
+class GameDifficulty(Choice):
+    """
+    Difficulty. This option determines how difficult the scores are to achieve.
+    Easy: for beginners. No luck required, just roll the dice and have fun. Lower final goal.
+    Medium: intended difficulty. If you play smart, you will finish the game without any trouble.
+    Hard: you will need to play smart and be lucky.
+    Extreme: really hard mode, which requires many brain wrinkles and insane luck. NOT RECOMMENDED FOR MULTIWORLDS.
+    """
+
+    display_name = "Game difficulty"
+    option_easy = 1
+    option_medium = 2
+    option_hard = 3
+    option_extreme = 4
+    default = 2
+
+
+class ScoreForLastCheck(Range):
+    """
+    The items in the item pool will always allow you to reach a score of 1000.
+    By default, the last check is also at a score of 1000.
+    However, you can set the score for the last check to be lower. This will make the game shorter and easier.
+    """
+
+    display_name = "Score for last check"
+    range_start = 500
+    range_end = 1000
+    default = 1000
+
+
+class ScoreForGoal(Range):
+    """
+    This option determines what score you need to reach to finish the game.
+    It cannot be higher than the score for the last check (if it is, it is changed automatically).
```

Comment:
Might be okay but "it is changed automatically", which of the two option mentionned is changed?